### DISCUSSION TITLE
Docker support

### DISCUSSION THREAD

**Discussion by trs91km** [sys4/netbox-plugin-dns#12] [Q&A]
This plugin support docker compose version of netbox installtion? please share the docs if available

**Comment by peteeckel** [comment]
Hi @trs91km, did you find the example in the [netbox-docker documentation](https://github.com/netbox-community/netbox-docker/wiki/Using-Netbox-Plugins)? I never tried it because I don't use Docker at all, but I don't think there should be any problems with NetBox DNS.

If you succeed it would be great when you could share your setup here.

**Comment by mikat-gh** [comment]
Hi,

We run our Netbox in Docker using netbox-docker.

Pretty much exactly as in the instructions linked, we just pin specific versions in `plugin_requirements.txt`, `Dockerfile-Plugins` and `docker-compose.override.yml` to avoid accidental version changes.
